charts/pega/README.md

```diff
@@ -232,15 +232,15 @@ Tier name     | Description
 ---           |---
 web           | Interactive, foreground processing nodes that are exposed to the load balancer. Pega recommends that these node use the node classification “WebUser” `nodetype`.
 batch         | Background processing nodes which handle workloads for non-interactive processing. Pega recommends that these node use the node classification “BackgroundProcessing” `nodetype`. These nodes should not be exposed to the load balancer.
-stream (Deprecated)        | From Pega Platform release 8.9 onwards, the use of the 'Stream' node classification is deprecated due to the shift towards an externalized Kafka configuration; for existing deployments where nodes that run an embedded deployment of Kafka and are not exposed to the load balancer, these nodes should continue using the node classification “Stream” nodetype.
+stream (Deprecated)        | For Pega Platform '23, the use of the 'Stream' node classification is deprecated; new deployments running version 8.8 and later should not use "Stream" nodes. New deployments connect to a Kafka service that you manage in your organization. For existing deployments using an embedded Kafka deployment which are not exposed to the deployment cluster load balancer, Pega will continue to support the "Stream node classification nodetype.
```

Comment:
almost...one missing quotation mark>>
Pega will continue to support the "Stream***"*** node classification nodetype.